src/frontend/magnify/src/components/design-system/Layout/Header/Avatar/ResponsiveLayoutHeaderAvatar.tsx

```diff
@@ -26,6 +28,16 @@ export const ResponsiveLayoutHeaderAvatar: FunctionComponent<ResponsiveLayoutHea
     },
   });
 
+  const getInitials = (): string => {
+    const user = authContext.user;
+    if (user) {
+      const splitName = user.name.split(' ');
+      const initials = splitName[0][0] + splitName[1][0];
+      return initials.toUpperCase();
+    }
+    return '';
+  };
```

Comment:
I think `useMemo` fits well this use case.

```suggestion
const initials = useMemo(() => {
    const { user } = authContext;
    if (user) {
      const splitName = user.name.split(' ');
      const initials = splitName[0][0] + splitName[1][0];
      return initials.toUpperCase();
    }
    return '';
  }, [authContext.user]);
```

Reply:
Yes !